examples/with-tailwindcss/src/layouts/main.astro

```diff
@@ -8,8 +9,9 @@ const { content } = Astro.props;
 		<meta name="viewport" content="width=device-width" />
 		<link rel="icon" type="image/svg+xml" href="/favicon.svg" />
 		<title>{content.title}</title>
+		<ViewTransitions />
 	</head>
-	<body>
+	<body transition:animate="morph">
```

Comment:
Do we want to remove the view transitions added in the examples before merging? These are used for create-astro templates

Reply:
I was thinking we wanted to promote the usage of this by having it in the examples. Will ask docs what they think.

Reply:
Docs thinks remove! :smile: